scripts/release_notes.rb

```diff
@@ -10,14 +10,19 @@
 
 URL = 'https://projects.theforeman.org'
 
-unless ARGV.size == 2
-  puts "Usage: #{$0} PROJECTS RELEASE_NAME"
+unless ARGV.size <= 2
+  puts "Usage: #{$0} PROJECTS RELEASE_NAME [markdown|asciidoc]"
   puts "Example: #{$0} foreman 1.18.0"
   exit 1
 end
 
 @project_names = ARGV[0].split(',')
 @current_release_name = ARGV[1]
+@output = ARGV[2] || "markdown"
+unless ["markdown", "asciidoc"].includes(@output)
```

Comment:
The method should be `include?` instead of `includes` ?

Reply:
Yeah sorry just tested this. Fixing.